fdbclient/tests/s3client_test.sh

```diff
@@ -128,35 +233,121 @@ if (( $# < 1 )) || (( $# > 2 )); then
 fi
 readonly build_dir="${1}"
 if [[ ! -d "${build_dir}" ]]; then
-  echo "ERROR: ${build_dir} is not a directory" >&2
+  err "${build_dir} is not a directory" >&2
   exit 1
 fi
 scratch_dir="${TMPDIR:-/tmp}"
 if (( $# == 2 )); then
   scratch_dir="${2}"
 fi
+readonly scratch_dir
 
-# Download seaweed.
-if ! weed_binary_path="$(download_weed "${scratch_dir}")"; then
-  echo "ERROR: failed download of weed binary." >&2
-  exit 1
-fi
-readonly weed_binary_path
-if ! create_weed_dir "${scratch_dir}"; then
-  echo "ERROR: failed create of the weed dir." >&2
-  exit 1
-fi
-log "Starting seaweed..."
-if ! s3_port=$(start_weed "${weed_binary_path}"); then
-  echo "ERROR: failed start of weed server." >&2
-  exit 1
+# Set host, bucket, and blob_credentials_file whether seaweed or s3.
+host=
+query_str=
+blob_credentials_file=
+path_prefix=
+if [[ -n "${OKTETO_NAMESPACE+x}" ]]; then
```

Comment:
May be better to explicitly print which branch is running by the test?

Reply:
Adding.